src/publish-scoped.js

```diff
@@ -60,6 +64,24 @@ function updateLockFiles(packageName) {
   updateLockFileIfExists('package-lock.json', packageName);
 }
 
+function verifyWixPackage(packageName) {
+  const result = execSync(`npm view ${packageName} publishConfig.registry`).toString();
```

Comment:
What happens if package is not found or registry is down currently? Will it return or break?

Reply:
This one is expensive command:

```
time npm view edit-contact publishConfig.registry
http://repo.dev.wix/artifactory/api/npm/npm-local/
npm view edit-contact publishConfig.registry  0.49s user 0.07s system 3% cpu 15.883 total
```

took about `15s` to complete

Reply:
```
✡ ◦ time npm view edit-contact@latest publishConfig.registry
http://repo.dev.wix/artifactory/api/npm/npm-local/
npm view edit-contact@latest publishConfig.registry  0.44s user 0.07s system 9% cpu 5.321 total

✡ ◦ time (curl --silent http://repo.dev.wix/artifactory/api/npm/npm-local/edit-contact/latest | jq '.publishConfig.registry')
"http://repo.dev.wix/artifactory/api/npm/npm-local/"
( curl --silent  | jq '.publishConfig.registry'; )  0.04s user 0.01s system 14% cpu 0.393 total
```

the second one (direct `curl`) is _alot_ faster. Not sure if this should be used though.. wdyt?

Reply:
`Exactly like what happens when publishing` - this is bad IMHO,
cause package `packageName` might not exist and `execSync` will throw an error breaking the build.

Reply:
I'm pushing some fixes, and here's the output for `verifyWixPackage`:

```javascript
> verifyWixPackage('this-package-surely-doesnt-exist-mantas!')
false
```

also, it shouldn't fail on unknown package, because the way I wrote it now 😄